sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuCompressedColumnVector.java

```diff
@@ -16,151 +16,66 @@
 
 package com.nvidia.spark.rapids;
 
-import ai.rapids.cudf.DType;
 import ai.rapids.cudf.DeviceMemoryBuffer;
-import com.nvidia.spark.rapids.format.ColumnMeta;
 import com.nvidia.spark.rapids.format.TableMeta;
-import org.apache.spark.sql.types.ArrayType;
-import org.apache.spark.sql.types.BinaryType;
-import org.apache.spark.sql.types.DataType;
-import org.apache.spark.sql.types.MapType;
-import org.apache.spark.sql.types.StructField;
-import org.apache.spark.sql.types.StructType;
 import org.apache.spark.sql.vectorized.ColumnVector;
 import org.apache.spark.sql.vectorized.ColumnarBatch;
 
+import static org.apache.spark.sql.types.DataTypes.NullType;
+
 /**
  * A GPU column vector that has been compressed. The columnar data within cannot
```

Comment:
I'm not sure that this is accurate.  It really takes the place of an entire table.